data-prepper-plugins/parquet-codecs/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+}
+
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(':data-prepper-api')
+    implementation 'org.apache.parquet:parquet-hadoop:1.12.0'
+    implementation 'org.apache.hadoop:hadoop-common:3.3.3'
+    implementation 'org.apache.parquet:parquet-avro:1.10.1'
```

Comment:
Do we need these Avro dependencies?

Reply:
Yes we do require these dependencies inside the test class to generate schema and to create random parquet stream. So `implementation` needs to be changed to `testImplementation`.